zadania_ruby/znajdz_index2.rb

```diff
@@ -0,0 +1,13 @@
+value = gets.to_i
```

Comment:
To nie do usunięcia jest lub przeniesienia do katalogu eksperyment?

Reply:
To jest wersja, którą ja napisałam. A find_index.rb - to co mi pokazywałeś. Ja przeniosę ten plik do eksperymentów.